pkg/scheduler/apis/config/types.go

```diff
@@ -55,6 +56,16 @@ type KubeSchedulerConfiguration struct {
 	// HardPodAffinitySymmetricWeight represents the weight of implicit PreferredDuringScheduling affinity rule, in the range 0-100.
 	HardPodAffinitySymmetricWeight int32
 
+	// TopologySpreadConstraints is the cluster-level configuration for
+	// topology-based pod spreading.
+	// Constraints are applied to pods that don't define any in their PodSpec.
+	// Label selectors must be empty, as they are deduced from the pods'
+	// membership in Services, Replication Controllers, Replica sets or Stateful
+	// sets.
+	// Omit to use the default: soft spreading among nodes and zones.
+	// Set to empty to disable.
+	TopologySpreadConstraints []corev1.TopologySpreadConstraint
```

Comment:
would naming  this `DefaultTopologySpreadConstraints` be better, to make it clearer this is only used if the pod doesn't specify topologySpreadConstraints?

Reply:
We considered this during the KEP discussion, and then we concluded this needed to be handled in the documentation.

The confusing part is that we would have a k8s-default value for the "default"

Reply:
ok, as long as it is well documented